packages/engine/Source/Scene/Model/ModelUtility.js

```diff
@@ -350,18 +350,20 @@ ModelUtility.supportedExtensions = {
   EXT_mesh_features: true,
   EXT_mesh_gpu_instancing: true,
   EXT_meshopt_compression: true,
+  EXT_primitive_voxels: true,
```

Comment:
Oops forgot to mention, could we add `EXT_implicit_cylinder_region` and `EXT_implicit_ellipsoid_region` to this list? They're not required for our implementation but they'll be included in generated cylinder / ellipsoid voxels

https://github.com/CesiumGS/glTF/tree/ext-primitive-voxels-revisions/extensions/2.0/Vendor/EXT_implicit_cylinder_region
https://github.com/CesiumGS/glTF/tree/ext-primitive-voxels-revisions/extensions/2.0/Vendor/EXT_implicit_ellipsoid_region